modules/ROOT/pages/clustering/setup/encryption.adoc

```diff
@@ -39,6 +39,25 @@ The generation of xref:security/ssl-framework.adoc#term-ssl-cryptographic-object
 It generally requires having a PKI with a xref:security/ssl-framework.adoc#term-ssl-certificate-authority[Certificate Authority (CA)] within the organization and they should be able to advise here.
 Note that the information in this manual relating to the PKI is mainly for illustrative purposes.
 
+[TIP]
+====
+If setting up intra-cluster encryption as part of a cluster configuration, do ensure that the certificates used on the cluster endpoint supports both Server and Client usage.
+This is because when connecting between the Neo4j servers for clustering, each server will use its own certificate to authenticate as a client on the connection to another server.
```

Comment:
```suggestion
This is because when connecting between the Neo4j servers for clustering, each server uses its own certificate to authenticate as a client on the connection to another server.
```